c_src/keyio.c

```diff
@@ -3,8 +3,13 @@
 #include <signal.h>
 #include <string.h>
 #include <unistd.h>
+#if __linux__
 #include <linux/input.h>
 #include <linux/uinput.h>
+#elif __FreeBSD__
```

Comment:
In my attempt to implement this, I needed:
```suggestion
#elif __FreeBSD__
#include <sys/ioctl.h>
```
I assume I didn't put that there without purpose though I'm not sure.